process/sync/baseSync.go

```diff
@@ -157,13 +157,13 @@ func (boot *baseBootstrap) processReceivedHeader(headerHandler data.HeaderHandle
 		log.Debug("forkDetector.AddHeader", "error", err.Error())
 	}
 
-	go boot.requestMiniBlocks(headerHandler.GetShardID(), headerHandler.GetNonce())
+	go boot.requestMiniBlocks(headerHash, headerHandler.GetNonce())
```

Comment:
You can send directly headerHanlder instead both parameters and we will avoid in the call method to access the pool again to get the same header

Reply:
done.